client/question/components/LearnerViewDirective.js

```diff
@@ -1214,22 +1209,12 @@ tie.directive('learnerView', [function() {
           EventHandlerService.createCodeResetEvent();
           $scope.autosaveCode();
 
-          // Resetting code starts a brand-new question session.
-          initLearnerViewDirective();
-        };
-
-        /**
-         * Clears the feedback in the window, and deletes the feedback
-         * from local storage for the current question.
-         */
-        $scope.resetFeedback = function() {
+          // Clear the code and feedback from localStorage, so that it is not
+          // retrieved in the subsequent initialization.
           SessionHistoryService.reset();
 
-          // Only add intro message if TIE is iframed so that the feedback
-          // window is not completely empty after resetting.
-          if (ParentPageService.isIframed()) {
-            SessionHistoryService.addIntroMessageBalloon();
-          }
```

Comment:
Is showing intro message done elsewhere or will be in a separate PR?

Reply:
It's done by session reset.